projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js

```diff
@@ -23,15 +24,23 @@ export default function EmailPreview( { isModalOpen, closeModal } ) {
 	const [ emailSending, setEmailSending ] = useState( false );
 	const [ errorMessage, setErrorMessage ] = useState( false );
 	const postId = useSelect( select => select( 'core/editor' ).getCurrentPostId() );
+	const { __unstableSaveForPreview } = useDispatch( editorStore );
 	const [ isSmall ] = useBreakpointMatch( 'sm' );
 	const { tracks } = useAnalytics();
 
-	const sendEmailPreview = () => {
+	const sendEmailPreview = async () => {
 		tracks.recordEvent( 'jetpack_send_email_preview', {
 			post_id: postId,
 		} );
 
 		setEmailSending( true );
+
+		// Save post revision so that we send what they see in the editor, and not what previous draft/revision might've saved
+		// Introduced at GB 16.3 at https://github.com/WordPress/gutenberg/pull/44971
```

Comment:
@Automattic/jetpack-crew in case you had advice on how to handle Gutenberg functions and backwards compatibility with older WPs in Jetpack; this isn't really issue for .com since we run the latest.

Method introduced in https://github.com/WordPress/gutenberg/pull/44971
It's here, and it's [contents are stable](https://github.com/WordPress/gutenberg/blob/37bca49efb6c8322d62adf642407c50f4270d9db/packages/editor/src/store/actions.js#L304-L322) so in theory I could just copy all that code here instead.

Reply:
Jetpack needs to support back to the version of the Gutenberg packages that is bundled in the previous WordPress release, which is currently 6.3.

I guess the case here is that 6.3 (and maybe 6.4 too?) doesn't have any version of the function at all, and you're wanting to copy-paste the implementation into Jetpack so the change you're making here still happens in 6.3? Be sure to include a comment indicating that the back-compat code can be removed when we drop support for the old version. Something along the lines of ``// @todo Remove the `if` check and `else` branch once WP 6.4 is the minimum supported version`` is fine, you don't have to get too fancy. Just be sure the comment clearly mentions an appropriate WordPress version so we can find it easily via a grep when we're making the PRs like #34210.

Or if it's ok that WP 6.3 just doesn't do whatever the new function does, that's fine with us too. Again though a comment mentioning the WP version when the `if` check can be removed would be useful.

Reply:
Done!